web/src/components/inspection/CreateInspectionPage.vue

```diff
@@ -117,7 +117,7 @@
           <v-btn class="mb-0" color="info" @click="addTaskClick">Add Hazard</v-btn>
           <v-spacer />
 
-          <v-btn class="mb-0" color="warning" to="/inspections">Complete Inspection</v-btn>
+          <v-btn class="mb-0" color="warning" @click="completeClick">Complete Inspection</v-btn>
```

Comment:
Prefer meaningful function names.
e.g. openConfirmationDialogAndGoToInspectionsPage
This will keep the function relevant if you change the trigger.
Function triggers generally change much for frequently than function effects.